src/ralph/backends/database/base.py

```diff
@@ -1,17 +1,18 @@
 """Base storage backend for Ralph"""
 
 from abc import ABC, abstractmethod
+from typing import Iterable
 
 
 class BaseDatabase(ABC):
-    """Base database backend interface"""
+    """Base database backend interface."""
 
     name = "base"
 
     @abstractmethod
     def get(self, chunk_size=10):
-        """Read chunk_size records and stream them to stdout"""
+        """Reads `chunk_size` records from the database and yields them."""
 
     @abstractmethod
-    def put(self, chunk_size=10, ignore_errors=False):
-        """Write chunk_size records from stdin"""
+    def put(self, stream: Iterable, chunk_size=10, ignore_errors=False):
```

Comment:
Shouldn't we be more specific?

```suggestion
    def put(
        self,
        stream: Union[BinaryIO, TextIO],
        chunk_size: int = 10,
        ignore_errors: bool = False,
    ):
```

Reply:
Good point) Thanks)